packages/components/src/StatusLabel/StatusLabel.css

```diff
@@ -14,7 +14,7 @@
   width: var(--statusLabel-icon-diameter);
   height: var(--statusLabel-icon-diameter);
```

Comment:
@chris-at-jobber Can we also make this 10px instead of 12px so it matches SG-1?

Reply:
✅